lecli/team_api.py

```diff
@@ -92,13 +93,13 @@ def create_team(name):
         response = requests.post(_url(), json=params, headers=headers)
         if response_utils.response_error(response):
             print 'Creating team failed, status code: %d' % response.status_code
```

Comment:
Sure, I'll update this and have a look to see if there's any more instances of this